modules/nf-core/cat/fastq/meta.yml

```diff
@@ -16,7 +17,7 @@ input:
         Groovy Map containing sample information
         e.g. [ id:'test', single_end:false ]
   - reads:
-      type: list
+      type: file
```

Comment:
is this supposed to be a map?

Reply:
I think file is correct here, because this type can also be multiple files as seen below